pennylane/templates/layers/basic_entangler.py

```diff
@@ -126,4 +126,4 @@ def circuit(weights):
     for layer in range(repeat):
 
         broadcast(unitary=rotation, pattern="single", wires=wires, parameters=weights[layer])
-        broadcast(unitary=CNOT, pattern="ring", wires=wires)
\ No newline at end of file
+        broadcast(unitary=CNOT, pattern="ring", wires=wires)
```

Comment:
```suggestion
        broadcast(unitary=CNOT, pattern="ring", wires=wires)
        
```

Not sure what this change is about? Did someone delete the new line at the end?

Reply:
Yeah, that was my fault, I was checking to see that all the changes from the earlier version of this PR were reverted and I accidentally deleted the end line.